requirements.txt

```diff
@@ -1,3 +1,3 @@
-build
+git+https://github.com/executablebooks/jupyter-book@v0.13
```

Comment:
FYI you can specify which version, branch, tag, commit you want to install using this kind of format

Reply:
Thanks @Remi-Gau! My purpose in manually going through these steps was to see if directly building from `pyproject.toml` would lead to a more stable behaviour as requirements of the executable book project are under active development, even for a fixed version of the book. 

But I guess, specifying it in the requirements.txt with a cut version would go through the same operations I manually inserted? It has been a while since I dealt with Python packaging stuff, happy to learn the differences between the two approaches, if there's any :)

Reply:
I am pretty sure it would be somewhat equivalent.
I have only recently started going for shallow dive in the python packaging wonderland, so I may be completely wrong. :shrug:

Reply:
Well, I am sure Jupyter Book will have some cool updates we'll like to implement at some point, maybe I'll figure out then :) For now, the repo is getting into good shape to be project worthy, thank you for your help!